src/codecs/imagequant/processor.ts

```diff
@@ -0,0 +1,43 @@
+import imagequant, { QuantizerModule } from '../../../codecs/imagequant/imagequant';
+import wasmUrl from '../../../codecs/imagequant/imagequant.wasm';
+import { QuantizeOptions } from './processor-meta';
+
+let emscriptenModule: Promise<QuantizerModule>;
```

Comment:
I changed how we do these types of files a bit. Instead of having a class, I just treat it as a module. The emscripten module is initialised on first use.